.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 dist: trusty
```

Comment:
`xenial` is available now: https://blog.travis-ci.com/2018-11-08-xenial-release

Reply:
Updated.